views/db/organizations/show.erb

```diff
@@ -0,0 +1,109 @@
+<% content_for :breadcrumbs  do %>
+<nav class="breadcrumb" aria-label="breadcrumbs">
+  <ul>
+    <li><a href="/">Home</a></li>
+    <li><a href="/db">Database</a></li>
+    <li><a href="/db/organizations">Organizations</a></li>
+  </ul>
+</nav>
+<% end %>
+
+<h1>Organization: <%=h @organization %></h1>
+
+<table class="table">
+  <tbody>
+    <tr>
+      <td><strong>Type:</strong></td>
+      <td><%=h @organization.type %></td>
+    </tr>
+
+    <tr>
+      <td><strong>Departments:</strong></td>
+      <td>
+        <% @organization.departments.each do |department| %>
+          <p><a href="/db/departments/<%=h department.id %>"><%=h department %></a></p>
```

Comment:
Should be named `/db/organization_departments/:id` (if we want to link directly to departments, without knowing their parent org ID) or `/db/organizations/:id/departments/:dept_id`.